nxt_editor/integration/maya/run_maya_graph.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/python
+#
+# Open or reference "sub_graphs" graph in Maya or Standalone and pass an nxt
+# graph, mayapy.exe path and parameters in the "_maya_sub_graph" node, and
+# this will run that graph in maya standalone
+#
+# Shell Example:
+# open a shell. Navigate to your bin/mayapy.exe file in your installation of
+# maya. Then run: mayapy.exe path/to/this/run_maya_graph.py -g path/to/nxt_graph/you_want_to_run.nxt
+#
+
+# command line arguments
+import argparse
+
+# Initialize parser
+parser = argparse.ArgumentParser(description="This is a cli for running the standalone maya")
+
+# Adding optional argument
+parser.add_argument("-g", "--graph_path", help="The path to the nxt graph you want to run.")
+parser.add_argument(
+    "-p",
+    "--parameters",
+    help="""The parameters you want to pass to the graph. Parameters are a string representing a 
+    dictionary.e.g. {'path/to/node.attr':'value'}""",
+),
+parser.add_argument(
+    "-s", "--start_node", help="The path to the nxt nofe inI the graph you want to run."
+)
+
+# Read arguments from command line
+args = parser.parse_args()
+
+
+if __name__ == "__main__":
+    if args.graph_path:
```

Comment:
Drop this statement in favor of making the arg required.

Reply:
I removed this in the latest push